src/main/kotlin/de/cyface/collector/storage/cloud/GoogleCloudStorageService.kt

```diff
@@ -182,15 +192,20 @@ class CloudStorageSubscriber<in T : Buffer>(
         this.subscription.cancel()
         cloudStorage.delete()
         this.error = t
+        logger.error("Error writing data!", t)
     }
 
     override fun onComplete() {
+        logger.debug("Completed writing data!")
         this.subscription.cancel()
     }
 
     override fun onNext(t: T) {
         // Write to Google Cloud
+        logger.debug("Streaming ${t.bytes.size} Bytes to Google Cloud Storage.")
+        streamedBytes += t.bytes.size
         cloudStorage.write(t.bytes)
-        this.subscription.request(1)
+        this.subscription.request(8192)
+        logger.debug("Progress: ${streamedBytes.toDouble()/totalBytes.toDouble() * 100.0} %")
     }
 }
```

Comment:
So was this the actual bug to be fixed? Or did I miss the actual fix somewhere else

Reply:
Also: CI fails

Reply:
> So was this the actual bug to be fixed? Or did I miss the actual fix somewhere else

I guess so.